src/core/trtserver.cc

```diff
@@ -610,6 +612,35 @@ TRTSERVER_InferenceRequestProviderSetInputData(
   return nullptr;  // Success
 }
 
+TRTSERVER_Error*
+TRTSERVER_InferenceRequestProviderSetSharedMemoryInputData(
```

Comment:
We don't need this function. Just use TRTSERVER_InferenceRequestProviderSetInputData along with TRTSERVER_ServerGetSharedMemoryInfo.

Reply:
TRTSERVER_InferenceRequestProviderSetInputData and TRTSERVER_InferenceRequestProviderSetSharedMemoryInputData require significantly different arguments. Should I still merge them?

Reply:
Merged. Can undo if needed.